src/Domain/InterfaceComputeTags.hpp

```diff
@@ -194,7 +133,8 @@ struct Slice : Interface<DirectionsTag, Tag>, db::ComputeTag {
     }
   }
   static std::string name() {
-    return "Interface<" + db::tag_name<Tag>() + ">";
+    return "Slice<" + db::tag_name<DirectionsTag>() + ", " +
+           db::tag_name<Tag>() + ">";
```

Comment:
This should use the name from the base class.

Reply:
I am not clear on what you are suggesting: 1) removing this `name()` function and therefore the one in `Tags::Interface` is used; or 2) changing something in this function.   I think you mean 1).